### ISSUE TITLE
Submission Format

### ISSUE THREAD

**Issue by yangdaowu** [dc3ea9f/vico_challenge_baseline#21]
Hello, I don’t know how to submit this step. How should I follow the speaker video, first frame and reference image generation test set video?

![image](https://github.com/dc3ea9f/vico_challenge_baseline/assets/64669901/9fed4810-3417-468f-bfe5-145c2527bdab)

I use the following code, but only generates the video of the training set.

![image](https://github.com/dc3ea9f/vico_challenge_baseline/assets/64669901/00e609bb-fc5a-4fea-9991-5f9241a1aab4)

**Comment by dc3ea9f** [comment]
This [issue](https://github.com/dc3ea9f/vico_challenge_baseline/issues/14) may help you do the inference with test set.

**Comment by yangdaowu** [comment]
I changed the path to the video path of the test set, but it still doesn't work

![image](https://github.com/dc3ea9f/vico_challenge_baseline/assets/64669901/bfbb795c-77d1-4444-b483-e55c7c7147bc)

**Comment by yangdaowu** [comment]
I changed the path to the video path of the test set, but it still doesn't work

**Comment by yangdaowu** [comment]
![image](https://github.com/dc3ea9f/vico_challenge_baseline/assets/64669901/91127286-cb4e-4dc5-995d-1fcc3d2e8752)

**Comment by dc3ea9f** [comment]
> https://github.com/dc3ea9f/vico_challenge_baseline/issues/14#issuecomment-1282043625
> you can generate a fake video by duplicating image to len(mfcc), and use that video to create vox_lmdb for visualization. For more details, please refer to [PIRender](https://github.com/RenYurui/PIRender).

First, you should generate a fake video, then, make predictions and render it.

**Comment by yangdaowu** [comment]
Sorry, bother again, how to generate fake videos?

**Comment by dc3ea9f** [comment]
Our videos are 30fps, and there is a mapping between the audio length and its mfcc feature length, get the mfcc feature length first, then you can just duplicate the first frame to the length of mfcc to generate the fake video.

**Comment by yangdaowu** [comment]
If possible, could you provide corresponding steps? Thank you.
